UI.tsv

```diff
@@ -1711,26 +1711,26 @@ UI_20150717_001710	CON : %s
 UI_20150717_001711	INT : %s
 UI_20150717_001712	DEX : %s
 UI_20150717_001713	SPR : %s
-UI_20150717_001714	{@st42b}Swordsman types can run by pressing the arrow key twice.{/}
-UI_20150717_001715	{@st42}Party Name and Description{/}
-UI_20150717_001716	{@st42}Insert Party Invitation Tag
+UI_20150729_001741	{@st42b}          Swordsman types can sprint by                   double-tapping the directional key.{/}
```

Comment:
Seems to have a lot of unnecessary spaces in this line of text, also it could be made clearer by specifying "a directional key" rather than "the directional key"